test/integration/basic_test.go

```diff
@@ -66,6 +68,21 @@ func (m *Minishift) shouldHaveAValidIPAddress() error {
 	return nil
 }
 
+func (m *Minishift) shouldHaveInstanceConfig() error {
```

Comment:
We can extract it to public util function. Also, it can't be function of MinishiftRunner